wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md

```diff
@@ -2,142 +2,158 @@
-\le \lambda_2(Q) \le \ldots \le \lambda_k(Q)$. Then we have  for small
-$t \geq 0$ and eigenvector $\ket{ x_j}$ corresponding to $\lambda_j \in
-\lambda(U)$ is given by  where $ S^{Q}\_{\_j(Q)}$;
+Assume that the eigenvalue $\lambda \in \lambda(U)$ is such that $r(\lambda) = k$. Let us define a matrix 
+$V(t)$ given by 
 
-6\. For each $j=1,\ldots,d$ we have  Moreover, 
+\begin{aligned}
+    V(t) = \exp(\ii tD_+) = \sum_{i=1}^d e^{\ii p_it} \proj{i} \in \mathcal{DU}\left(\mathbb{C}^d\right),
+    \quad t \ge 0.
+\end{aligned}
 
-Proof of this therorem you can see in {% cite kukulski2020 %}.
+Let $\lambda(t):=\lambda(UV(t))$ and let every $\lambda_j(t) \in \lambda(t)$ corresponds to
+eigenvector $\ket{x_j(t)}$. Assume that $\lambda_{1}(t), \ldots, \lambda_{k}(t)$ are
+such eigenvalues that $\lambda_{j}(t) \to \lambda$, as $t \to 0$. Then:
 
-Intuitively speaking, this theorem gives us equations which one can use
-to predict behaviour of $W(UV(t))$. Observe the postulate $(6)$ fully
-determines the movement of the spectrum. However, this is a theoretical
-statement and in practice determining the function $t $ is a numerically
-complex task. The postulates $(1)-(5)$ play a key role in numerical
-calculations of $W(UV(t))$. The most important fact comes from $(3)$
+1. If $ \min\\{ \sum_{i=1}^d p_i |\braket{i}{x}|^2: \ket{x} \in S^{U}_{\lambda} \\}=0$, then $\lambda$ is an
+eigenvalue of $UV(t)$.
+
+2. If $\| \{p_i: p_i >0 \}\|=l\<k$, then $\lambda$ is an eigenvalue of
+$UV(t)$ and $r(\lambda) \ge k-l$.
+
+3. Each eigenvalue of product $UV(t)$ moves counterclockwise or stays
+in the initial position as parameter $t$ increases.
+
+4. If $k=1$, then 
+\begin{equation}
+\lambda_1(t) \simeq \lambda \exp \left( \ii t \sum_{i=1}^d p_i | \braket{i}{x_1} |^2 \right)
+\end{equation}
+for small $t \ge 0$.
+
+5. Let $Q \coloneqq I_{U,\lambda}^\dagger D_+ I_{U,\lambda}$ and $\lambda_1(Q) \le \lambda_2(Q) \le \ldots 
+\le \lambda_k(Q)$. Then we have 
+\begin{equation}
+\lambda_j(t) \simeq \lambda \exp(\ii \lambda_j(Q)t) 
+\end{equation}
+for small $t \ge 0$ and eigenvector $\ket{x_j}$ corresponding to $\lambda_j \in \lambda(U)$ is 
+given by
+\begin{equation}
+\ket{x_j} = I_{U, \lambda} \ket{v_j},
+\end{equation}
+where $\ket{v_j} \in S^{Q}_{\lambda_j(Q)}$.
+
+6. For each $j=1,\ldots,d$ we have  
+\begin{equation}
+\frac{\partial}{\partial t} \lambda_j(t) = \ii \lambda_j(t) \sum_{i = 1}^d p_i | \braket{i}{x_j(t)} | ^2.
+\end{equation}
+
+Moreover,
+
+\begin{equation}
+\sum_{j=1}^{d} \left| \frac{\partial}{\partial t} \lambda_j(t) \right| = 1.
+\end{equation}
+
+Intuitively speaking, this theorem gives us hints, which one can use
+to predict a behaviour of $W(UV(t))$. Observe that, the postulate $6.$ fully
+determines the movenment of the spectrum. However, this is a theoretical
+statement and in practice, determining the function $t \mapsto \ket{x_j(t)}$ is a numerically
+complex task. The postulates from $1.$ to $5.$ play a key role in numerical
```

Comment:
I wouldn't call these points postulates.

Reply:
We are using the  terminology from https://doi.org/10.1007/978-3-030-50433-5_48